installation/aws/VmClarity.cfn

```diff
@@ -148,102 +148,293 @@ Resources:
+                    echo "DB_NAME=${ExternalDBName}" >> /etc/vmclarity/apiserver.env
+                    echo "DB_USER=${ExternalDBUsername}" >> /etc/vmclarity/apiserver.env
+                    echo "DB_PASS=${ExternalDBPassword}" >> /etc/vmclarity/apiserver.env
+                    echo "DB_HOST=${ExternalDBHost}" >> /etc/vmclarity/apiserver.env
+                    echo "DB_PORT_NUMBER=${ExternalDBPort}" >> /etc/vmclarity/apiserver.env
                   elif [ "${DatabaseToUse}" == "SQLite" ]; then
                     # Configure the VMClarity backend to use the SQLite DB
                     # driver and configure the storage location so that it
                     # persists.
-                    echo "DATABASE_DRIVER=LOCAL" >> /etc/vmclarity/config.env
-                    echo "LOCAL_DB_PATH=/data/vmclarity.db" >> /etc/vmclarity/config.env
+                    echo "DATABASE_DRIVER=LOCAL" > /etc/vmclarity/apiserver.env
+                    echo "LOCAL_DB_PATH=/data/vmclarity.db" >> /etc/vmclarity/apiserver.env
                   fi
 
-                  # Replace anywhere in the config.env __BACKEND_REST_HOST__
+                  # Replace anywhere in the config.env __CONTROLPLANE_HOST__
                   # with the local ipv4 IP address of the VMClarity server.
                   local_ip_address="$(curl http://169.254.169.254/latest/meta-data/local-ipv4)"
-                  sed -i "s/__BACKEND_REST_HOST__/${!local_ip_address}/" /etc/vmclarity/config.env
+                  sed -i "s/__CONTROLPLANE_HOST__/${!local_ip_address}/" /etc/vmclarity/orchestrator.env
+
+                  # Reload the systemd daemon to ensure that the VMClarity unit
+                  # has been detected.
+                  systemctl daemon-reload
+
+                  # Create directory required for grype-server
+                  /usr/bin/mkdir -p /opt/grype-server
+                  /usr/bin/chown -R 1000:1000 /opt/grype-server
+
+                  # Create directory required for vmclarity apiserver
+                  /usr/bin/mkdir -p /opt/vmclarity
+
+                  # Create directory for exploit db server
+                  /usr/bin/mkdir -p /opt/exploits
+
+                  # Create directory for trivy server
+                  /usr/bin/mkdir -p /opt/trivy-server
 
                   # Enable and start/restart VMClarity backend
                   systemctl enable vmclarity.service
                   systemctl restart vmclarity.service
               mode: "000744"
-            "/etc/vmclarity/config.env":
+
+            "/etc/vmclarity/vmclarity.yaml":
+              content:
+                Fn::Sub:
+                  - |
+                    version: '3'
+
+                    services:
+                      apiserver:
+                        image: ${APIServerContainerImage}
+                        command:
+                          - run
+                          - --log-level
+                          - info
```

Comment:
maybe not related but, don't we want logging level to be configurable?

Reply:
I think thats probably a good enhancement, I'll open a ticket.

Reply:
https://github.com/openclarity/vmclarity/issues/520